packages/storybook/stories/CallComposite/components/IncomingCallAlert.tsx

```diff
@@ -0,0 +1,100 @@
+import { DefaultButton, DefaultPalette, mergeStyles, Persona, PersonaSize, Stack, useTheme } from '@fluentui/react';
+import { Call20Filled, CallEnd20Filled } from '@fluentui/react-icons';
+import React from 'react';
+
+const palette = DefaultPalette;
```

Comment:
are opposed to exporting this component in the package until we get designs to what this might look like?

Reply:
Good point, I could add this component into SimpleInboundHomeScreen instead of having a separate component. 
That would resolve any issues with exporting this specific component.